dockerfiles/Dockerfile.browser-v6

```diff
@@ -0,0 +1,51 @@
+# CI test image for unit/lint/type tests
+FROM node:14-alpine as browser-feature-builder
```

Comment:
just a question - what's the purpose of aliasing the base image?

Reply:
See L50

Reply:
ahh! got it. thanks!

Reply:
There's not really one, except for clarity.